azure-pipelines.yml

```
@@ -66,7 +66,7 @@ jobs:
         inputs:
           packageType: 'sdk'
           version: '$(NET_VERSION)'
-          includePreviewVersions: false
+          includePreviewVersions: true
```

Change to false before merging
```suggestion
          includePreviewVersions: false
```

---

false